GetContact/index.ts

```diff
@@ -1,24 +1,24 @@
-import { AzureFunction, Context, HttpRequest } from "@azure/functions"
-import { getAllTodoItems } from "../DataAccess/contactList";
+import {AzureFunction, Context, HttpRequest} from '@azure/functions';
+import {getContact} from '../DataAccess/contactList';
 
-const httpTrigger: AzureFunction = async function (context: Context, req: HttpRequest): Promise<void> {
-    context.log.info('Get all todo items started');
-    
-    const cont = await getAllTodoItems("2021-10-08T11:08:04.340Z45308239");
-    context.log.info(cont);
-    if(cont && cont.length > 0) {
-        context.res = {
-            status: 200,
-            body: cont
-        };
-    } else {
-        context.res = {
-            status: 204
-        };
-    }
-
-    context.log.info('Get all todo items completed.');
+const httpTrigger: AzureFunction = async function (
+  context: Context,
+  req: HttpRequest
+): Promise<void> {
+  context.log.info(req);
+  const contact = await getContact(req.query.id, req.query.userId);

```

Comment:
perhaps getContact should not be returning an array?

Reply:
`contact` should be `contacts` if you're expecting an array.